CHANGES.txt

```diff
@@ -5,6 +5,8 @@
  - Removed the disable_impressions configuration param.
  - Fixed an issue about Event properties were not sent to be.
  - Added ip_addresses_enabled to enable/disable sending MachineName and MachineIP headers when data is posted to Split Servers.
+ - Fixed an issue about attributes in matchers.
+ - Fixed an issue about find latencies.
```

Comment:
The "find latencies" does not mean much externally. Maybe describe the impact? Fixed an issue about trying to access a null property when sending telemetry metrics.